the-basic-mq/python/README.md

```diff
@@ -0,0 +1,203 @@
+# The Basic MQ
+
+![architecture](img/the-basic-mq-arch.png)
+
+This is an example cdk stack to deploy [static custom domain endpoints with Amazon MQ](https://aws.amazon.com/blogs/compute/creating-static-custom-domain-endpoints-with-amazon-mq/)  from Rachel Richardson.
+
+In this example we have private Amazon MQ brokers behind an internet-facing network load balancer endpoint using a subdomain.
+
+## Pre Requirements
+
+This pattern requires you to have a Route53 Hosted Zone already in your account so that you can assign a public URL to your NLB.
+
+After you setup a hosted zone, you can get the id from the console and replace the value for 'hosted_zone_id' in the cdk stack. Then you need to replace 'zone_name' and 'subdomain_name' with the url you desire in that hosted zone.
+
+Also, you have to have a certificate for the subdomain before you deploy the stack (to avoid exceeding [ACM yearly certificate limit](https://github.com/aws/aws-cdk/issues/5889)).
+
+so if your hosted zone id is 1234 and you want your public url to be mq.cdkpatterns.com you would do:
+
+```python
+# Paste Hosted zone ID from Route53 console 'Hosted zone details'
+hosted_zone_id = '1234'
+
+# If zone_name = 'cdkexample.com' and subdomain_name = 'iot', you can connect to the broker by 'iot.cdkexample.com'.
+zone_name = 'cdkpatterns.com'
+subdomain_name = 'mq'
+
+# Request and issue a certificate for the subdomain (iot.cdkexample.com in this example) beforehand, and paste ARN.
+cert_arn = 'arn:aws:acm:us-east-1:2XXXXXX9:certificate/exxxx8-xxxx-xxxx-xxxx-fxxxxxxxx9'
```

Comment:
The TS version creates the cert for you, what made you change it?

Reply:
With `DnsValidatedCertificate` CDK creates a different cert for the same domain when you delete and re-deploy the stack.  You can issue 20 certs per year (https://github.com/aws/aws-cdk/issues/5889) on new AWS accounts, and I thought it is better to manually issue a cert and use it by ARN.

Reply:
Is it worth adding a comment to that effect? Linking to that issue

Reply:
I agree, I had better add some comment on it.